#VRML_SIM R2019a utf8
WorldInfo {
}
Viewpoint {
  orientation 1 0 0 4.71238898038469
  position 0.14495798123380554 2.684476953310028 0.00731915780962712
}
TexturedBackground {
}
TexturedBackgroundLight {
}
Floor {
}
Robot {
  translation 0 0.075 0
  children [
    DEF SH_BODY Group {
      children [
        Shape {
          appearance PBRAppearance {
            baseColor 0.8 0 0
            roughness 1
            metalness 0
          }
          geometry Box {
            size 0.25 0.08 0.1
          }
        }
        Transform {
          translation 0.1075 -0.0575 0
          children [
            Shape {
              appearance PBRAppearance {
                baseColor 0.9882352941176471 0.9137254901960784 0.30980392156862746
                roughness 1
                metalness 0
              }
              geometry Sphere {
                radius 0.0175
              }
            }
          ]
        }
      ]
    }
    HingeJoint {
      jointParameters HingeJointParameters {
        axis 0 0 1
        anchor -0.05 0 0.065
      }
      device [
        RotationalMotor {
          name "motor_right"
          maxVelocity 52.36
          maxTorque 0.5932
        }
        PositionSensor {
          name "position_right"
          resolution 0.005235
        }
      ]
      endPoint Solid {
        translation -0.05 0 0.065
        rotation 1 0 0 1.5707996938995745
        children [
          Shape {
            appearance DEF BLACK PBRAppearance {
              baseColor 0 0 0
              roughness 1
              metalness 0
            }
            geometry DEF WHEEL1 Cylinder {
              height 0.03
              radius 0.075
            }
          }
        ]
        boundingObject USE WHEEL1
        physics Physics {
        }
      }
    }
    HingeJoint {
      jointParameters HingeJointParameters {
        axis 0 0 1
        anchor -0.05 0 -0.065
      }
      device [
        RotationalMotor {
          name "motor_left"
          maxVelocity 52.36
          maxTorque 0.5932
        }
        PositionSensor {
          name "position_left"
          resolution 0.005235
        }
      ]
      endPoint Solid {
        translation -0.05 0 -0.065
        rotation 1 0 0 1.5707996938995745
        children [
          Shape {
            appearance USE BLACK
            geometry USE WHEEL1
          }
        ]
        name "solid(1)"
        boundingObject USE WHEEL1
        physics Physics {
        }
      }
    }
    DistanceSensor {
      translation 0.125 0 0
      children [
        DEF SENS Shape {
          appearance PBRAppearance {
            baseColor 0.20392156862745098 0.396078431372549 0.6431372549019608
            roughness 1
            metalness 0
          }
          geometry Box {
            size 0.01 0.01 0.01
          }
        }
      ]
      name "distance_sensor"
      boundingObject USE SENS
      lookupTable [
        0 0 0
        0.02 0 0
        5 1660 0
      ]
      resolution 1
    }
  ]
  boundingObject USE SH_BODY
  physics Physics {
  }
  controller "crazybot_Pieiro_Lazaro"
}
